lib/versioning/pep440/range.ts

```diff
@@ -5,29 +5,78 @@ import { logger } from '../../logger';
 import { regEx } from '../../util/regex';
 import type { NewValueConfig } from '../types';
 
+// eslint-disable-next-line typescript-enum/no-enum
```

Comment:
I think you can update the documentation by changing the `allowedValues` property for the `rangeStrategy` option:

https://github.com/renovatebot/renovate/blob/15c393f79115c0a5fe6429396c5a9e51127ffc67/lib/config/options/index.ts#L1168-L1184

---

@viceice do you know where we keep the source for the autogenerated stuff on this page? https://docs.renovatebot.com/modules/versioning/#pep440-versioning

Reply:
It's generated from the exports of `index.ts`

Reply:
so which approach are we going for?
widen -> extend (1 file change)
or the other way around
extend -> widen? (change in 11 files)

Reply:
see pr #14432